test/framework/src/main/java/org/elasticsearch/test/InternalTestCluster.java

```diff
@@ -777,6 +808,10 @@ public int nodeAndClientId() {
             return nodeAndClientId;
         }
 
+        public boolean isMasterNode() {
```

Comment:
yeah - this is always a good one. It will take a much bigger change. For example, the setting is called `NODE_MASTER_SETTING` - i opted to use the short lingo